### DISCUSSION TITLE
Drawer Lagging. Needs a Performance boost on Mobile Devices

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [saadeghi/daisyui#2537] [Q&A]
Hi,

I have the problem that my drawer is not really responsive on my mobile device (Iphone 13 so not old hardware ^^).

First I thought it must be my own drawer and how I code it but I also tested the navigation drawer at https://daisyui.com/ and I found the same problem. As soon as i open the drawer it takes about 500-800ms to actually open it.

Is it possible to boost the responsiveness of the drawer?

**Comment by MuslemRahimi** [comment]
Would be nice if someone could address this issue

**Reply by saadeghi** [reply]
I couldn't reproduce this issue ony devices or even simulator. It has no delay for me.

Is it laggy if the content is too much or it's also laggy with an empty drawer?

Can you try disabling the CSS transition to see if it helps?

I can't help with this unless I find a device where I can see the issue.

**Comment by NikarashiHatsu** [comment]
It's Apple being Apple.
